heron/packing/src/java/com/twitter/heron/packing/roundrobin/ResourceCompliantRRPacking.java

```diff
@@ -34,9 +34,6 @@
 import com.twitter.heron.spi.packing.Resource;
 import com.twitter.heron.spi.utils.TopologyUtils;
 
-import static com.twitter.heron.api.Config.TOPOLOGY_CONTAINER_CPU_REQUESTED;
```

Comment:
IMO the code is less verbose and easier to read when using these static imports and not the fully qualified param below.